tests/test_wave_01.py

```diff
@@ -119,12 +119,14 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
```

Comment:
This test is OK as a quick check, but as a minor test design thing, I would recommend checking all the keys of the movie that got watched, not just the title. The easiest way to do this would likely to save the expected dictionary and use `==` to compare the entire dictionary, but admittedly, the arrange part of this code that we wrote would have to change...